cfme/tests/cloud/test_cloud_timelines.py

```diff
@@ -272,17 +268,17 @@ def test_cloud_timeline_create_event(new_instance, soft_assert, azone):
         assignee: jdupuy
         initialEstimate: 1/4h
     """
-    targets = (new_instance, new_instance.provider, azone)
+    if BZ(1670550).blocks:
+        targets = (new_instance, )
```

Comment:
Optional:
I know this is `black` but not that much strictness :)

Reply:
This formatting is actually necessary for the test, since `_check_timelines` requires `targets` to be an iterable object.